frontend/src/components/common/LogViewer.tsx

```diff
@@ -215,6 +222,13 @@ export function LogViewer(props: LogViewerProps) {
   );
 }
 
+// clears logs for pod
+function clearPodLogs(xtermRef: React.MutableRefObject<XTerminal | null>) {
+  xtermRef.current?.clear();
+  // keeping this comment if logs dont print after clear
+  // xtermRef.current?.write(getJointLogs());
```

Comment:
This comment should be removed. If the logs do not print after clear, then it's a problem we need to solve.